lib/mix/tasks/compile.mavlink.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Mix.Tasks.Compile.Mavlink do
+  use Mix.Task
+
+  def run(_args) do
+    HTTPoison.start()
+
```

Comment:
This is boilerplatey and can impress my peers with clever code if we want to take this approach of pulling things down in the compilation step.